git_machete/tests/functional/test_machete.py

```diff
@@ -193,25 +193,25 @@ def adapt(s):
 class MacheteTester(unittest.TestCase):
 
     @staticmethod
-    def launch_command(*args):
+    def launch_command(*args: str) -> str:
         orig_out = sys.stdout
         out = io.StringIO()
         sys.stdout = out
         try:
-            cmd.launch(args)
+            cmd.launch(args)  # type: ignore
```

Comment:
Hmm why `ignore` needed here? not that it matters much since it's test code anyway

Reply:
given tuple, required list